Gems/Atom/RPI/Code/Source/RPI.Public/Image/StreamingImage.cpp

```diff
@@ -170,7 +206,10 @@ namespace AZ
 
                     // We want to store off the id, not the AssetData instance. This simplifies the fetch / evict logic which
                     // can do more strict assertions.
-                    m_mipChains.push_back(Data::Asset<ImageMipChainAsset>(assetId, azrtti_typeid<ImageMipChainAsset>()));
+                    if (mipChainIndex >= highestMipChain)
+                        m_mipChains.push_back(Data::Asset<ImageMipChainAsset>(assetId, azrtti_typeid<ImageMipChainAsset>()));
+                    else
+                        m_mipChains.push_back(Data::Asset<ImageMipChainAsset>{});
```

Comment:
o3de style: o3de style guidelines require `{` `}`
```suggestion
                    if (mipChainIndex >= highestMipChain)
                    {
                        m_mipChains.push_back(Data::Asset<ImageMipChainAsset>(assetId, azrtti_typeid<ImageMipChainAsset>()));
                    }
                    else
                    {
                        m_mipChains.push_back(Data::Asset<ImageMipChainAsset>{});
                    }
```